xmodule/static_content.py

```diff
@@ -147,10 +147,8 @@ def _write_styles(selector, output_root, classes, css_attribute):
         for class_ in classes:
             css_imports[class_].add(fragment_name)
 
-    module_styles_lines = [
-        "@import 'bourbon/bourbon';",
-        "@import 'lms/theme/variables';",
```

Comment:
no, that's false, probably one of them uses `lms/theme/variables`, however in this context it's hard to know which one, because  the file `_module-styles.scss` is imported in a bigger file ([_build-course.scss](https://github.com/openedx/edx-platform/blob/master/lms/static/sass/_build-course.scss) or [_build-v1.scss](https://github.com/openedx/edx-platform/blob/master/cms/static/sass/_build-v1.scss)) that already import that.

if you check this [list](https://github.com/openedx/edx-platform/blob/d3f82fafc426f77dbab7706b6536e3c240bd2a3d/xmodule/static_content.py#L93) after decoupling you will find that more of one import is necessary

Reply:
Got it. Do you want to merge this PR first? If so, when you rebase [the decoupling PR](https://github.com/openedx/edx-platform/pull/32018), do you think you will put `@import 'lms/theme/variables';` back for some files?

Reply:
yes, I'd like to merge this one first,  in that way I can add just the required imports in the specific files in this [PR](https://github.com/openedx/edx-platform/pull/32018)